control-plane-addon/src/index.ts

```diff
@@ -81,26 +70,30 @@ type KeptnControlPlaneParams = {
     ingressSecretName: string,
 }
 
-const defaultKeptnControlPlaneParams: KeptnControlPlaneParams = {
+export const defaultProps: HelmAddOnProps & KeptnControlPlaneAddOnProps = {
+    name: 'keptn',
+    namespace: "keptn",
+    repository: "https://storage.googleapis.com/keptn-installer",
+    version: "0.11.4",
+    release: "ssp-addon-keptn-cp",
```

Comment:
Release name should be "keptn" else it would not work with the helm annotations for the secrets.